src/Message/UpdateCustomerAddressRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Guzzle\Http\Message\RequestInterface;
+use Omnipay\Common\Exception\InvalidRequestException;
+
+final class UpdateCustomerAddressRequest extends AbstractOrderRequest
+{
+    /**
+     * @return null
+     *
+     * @throws InvalidRequestException
+     */
+    public function getData()
+    {
+        $this->validate('transactionReference');
+
+        return null;
```

Comment:
Should there not be some data you wish to update? Seems like a rather pointless request otherwise